lib/npm-auditer.test.js

```diff
@@ -0,0 +1,156 @@
+  });
+  it('does not report critical severity if it set to false', async () => {
+    const summary = await audit(
+      config({
+        dir: testDir('npm-critical'),
+        levels: { critical: false },
+      }),
+      summary => summary
+    );
+    expect(summary).to.eql({
+      whitelistedModulesFound: [],
+      whitelistedAdvisoriesFound: [],
+      failedLevelsFound: [],
+      advisoriesFound: [],
+    });
+  });
+  it('reports high severity', async () => {
+    const summary = await audit(
+      config({
+        dir: testDir('npm-high'),
+        levels: { high: true },
+      }),
+      summary => summary
+    );
+    expect(summary).to.eql({
+      whitelistedModulesFound: [],
+      whitelistedAdvisoriesFound: [],
+      failedLevelsFound: ['high'],
+      advisoriesFound: [690],
+    });
+  });
+  it('reports moderate severity', async () => {
+    const summary = await audit(
+      config({
+        dir: testDir('npm-moderate'),
+        levels: { moderate: true },
+      }),
+      summary => summary
+    );
+    expect(summary).to.eql({
+      whitelistedModulesFound: [],
+      whitelistedAdvisoriesFound: [],
+      failedLevelsFound: ['moderate'],
+      advisoriesFound: [658],
+    });
+  });
+  it('does not report moderate severity if it set to false', async () => {
+    const summary = await audit(
+      config({
+        dir: testDir('npm-moderate'),
+        levels: { moderate: false },
+      }),
+      summary => summary
+    );
+    expect(summary).to.eql({
+      whitelistedModulesFound: [],
+      whitelistedAdvisoriesFound: [],
+      failedLevelsFound: [],
+      advisoriesFound: [],
+    });
+  });
+  it('ignores an advisory if it is whitelisted', async () => {
+    const summary = await audit(
+      config({
+        dir: testDir('npm-moderate'),
+        levels: { moderate: true },
+        advisories: [658],
+      }),
+      summary => summary
+    );
+    expect(summary).to.eql({
+      whitelistedModulesFound: [],
+      whitelistedAdvisoriesFound: [658],
+      failedLevelsFound: [],
+      advisoriesFound: [],
+    });
+  });
+  it('does not ignore an advisory that is not whitelisted', async () => {
+    const summary = await audit(
+      config({
+        dir: testDir('npm-moderate'),
+        levels: { moderate: true },
+        advisories: [659],
+      }),
+      summary => summary
+    );
+    expect(summary).to.eql({
+      whitelistedModulesFound: [],
+      whitelistedAdvisoriesFound: [],
+      failedLevelsFound: ['moderate'],
+      advisoriesFound: [658],
+    });
+  });
+  it('reports low severity', async () => {
+    const summary = await audit(
+      config({
+        dir: testDir('npm-low'),
+        levels: { low: true },
+      }),
+      summary => summary
+    );
+    expect(summary).to.eql({
+      whitelistedModulesFound: [],
+      whitelistedAdvisoriesFound: [],
+      failedLevelsFound: ['low'],
+      advisoriesFound: [577],
+    });
+  });
+  it('none', async () => {
```

Comment:
Looks like the node 6 build is failing since `async` functions aren't supported there. This is only in the tests, but I guess there's nothing we can do to test node 6 with a higher language level just in the test file. So I'd suggest to convert these to callback style, assuming that we're intending to support node 6 @quinnturner

Reply:
Done